components/ui/faq-accordion.tsx

```
@@ -130,8 +130,27 @@ const faqs = [
+        "@type": "Answer",
+        text: faq.answer,
+      },
+    })),
+  }
+
   return (
     <div className="max-w-3xl w-full mx-auto">
+      <script
+        type="application/ld+json"
+        // Static strings defined in this file; safe to stringify directly
+        dangerouslySetInnerHTML={{ __html: JSON.stringify(faqSchema) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)